src/main/java/core/basesyntax/figures/FigureSupplier.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.color.ColorSuplier;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
+    private static final int figures = 5;
+    private static final int maxSize = 99;
+    private static final String defaultColor = "white";
+    private static final int defaultRadius = 10;
+
+    public Figure getRandomFigure() {
+        return getFigure();
+    }
+
+    private Figure getFigure() {
```

Comment:
Not fixed, replace code to getRandomFigure method

Reply:
not fixed. It doens't make sense to have a public method that does one single call to the private method. just move logic there